packages/sewing-kit-koa/src/tests/middleware.test.ts

```diff
@@ -48,13 +45,32 @@ describe('middleware', () => {
     );
   });
 
-  it('accespts a custom value for the manifest path', async () => {
+  it('accepts a custom value for the manifest path', async () => {
     const context = createMockContext();
     const manifestPath = 'path/to/manifest';
     await middleware({manifestPath})(context, () => Promise.resolve());
     expect(getAssets(context)).toHaveProperty('manifestPath', manifestPath);
   });
 
+  it('accepts a assetPrefix callback and runs the callback', async () => {
+    const context = createMockContext();
+    const customAssetPrefix = 'https://cdn.some.site/path';
+
+    await middleware({
+      assetPrefixCallback: ctx => {
+        return customAssetPrefix;
+      },
+    })(context, () => Promise.resolve());
+
+    expect(getAssets(context)).toHaveProperty('assetPrefix', customAssetPrefix);
+
+    await middleware({
+      assetPrefixCallback: ctx => {},
+    })(context, () => Promise.resolve());
+
+    expect(getAssets(context)).toHaveProperty('assetPrefix', webpackAssetUrl);
+  });
```

Comment:
I think we probably want a test that verifies that `assetPrefix` is called with `context` (we don't currently use that feature anywhere in the tests, so it's not technically verified)